src/pages-and-resources/discussions/data/api.js

```diff
@@ -90,17 +90,37 @@ export function getAppConfig(courseId, appId) {
       app = yellowdigApp;
       break;
     default:
-      app = edXForumsApp;
+      app = legacyEdXDiscussions;
+  }
+
+  let appConfig = {
+    id: 'appConfig1',
+    consumerSecret: 'its-a-secret-to-everybody',
+    consumerKey: 'abc123',
+    launchUrl: 'https://localhost/launch',
+  };
+
+  if (appId === 'edx-discussions') {
+    appConfig = {
+      id: 'appConfig2',
```

Comment:
This seems opaque to me; what's this used for and/or what's the difference?

Reply:
This is just crappy test data - the appConfig ID will ultimately be whatever identifier we send up from the backend - probably just the PK in the database table.